layout/templates/textSection.mustache

```diff
@@ -0,0 +1,4 @@
+<div class="viz-text">
```

Comment:
Can we put this <div class="viz-text"> in the printAll template in vizlab? That way, we don't need to make a textSection mustache for each viz. We still could...but we wouldn't have to.

Reply:
No clue, can we?

Reply:
yes, i think laura has it. modify this: https://github.com/USGS-VIZLAB/vizlab/blob/master/inst/templates/printall.mustache. also helps with future vizzies already having divs with `viz-text` class by default so that the styling added in https://github.com/USGS-VIZLAB/vizlab/pull/362 will get applied